uikit/src/main/java/com/midtrans/sdk/uikit/activities/CreditCardFlowActivity.java

```diff
@@ -867,4 +881,38 @@ private void showBankPointsFragment(float balance, String bankType) {
         getSupportFragmentManager().beginTransaction().replace(R.id.card_container, fragment).addToBackStack("").commit();
     }
 
+    public boolean isClickPayment() {
```

Comment:
Ok then, but can we have token only support in v1 SDK?
I want to use it on Prism SDK :))